js/plates-model/plate.js

```diff
@@ -79,6 +84,49 @@ export default class Plate extends PlateBase {
     return totalTorque.applyMatrix3(this.invMomentOfInertia)
   }
 
+  updateCenter() {
+    const safeFields = {}
+    let safeSum = new THREE.Vector3()
```

Comment:
This could be a `const`. Note that reference is never changed, only underlying object. It's quite a common style in JS to use `const` that actually can be. And it's used within this project, so I'd keep doing it. Not saying that this convention provides much value, but just to be consistent. :wink:

Reply:
Whoops! I actually was changing the reference before, I didn't realize I could mutate Vectors. Fixed now, nice catch